index.js

```diff
@@ -43,4 +51,6 @@ function clipboardCopy (text) {
   document.body.removeChild(iframe)
 
   return success
+    ? Promise.resolve()
+    : Promise.reject() // eslint-disable-line prefer-promise-reject-errors
```

Comment:
Hmm, it seems a bit odd to reject without an error? How do you feel about `Promise.reject(new Error('Failed writing to clipboard'))`?

Reply:
I was matching the browser API. Even if I reject with an error here, the user can't rely on that, since if the Async Clipboard API is available then it may reject without an error.

Reply:
Aha, interesting... I wonder why they would standardize around that 🤔 

Do you have a link where I can read about this? I tried searching but to me, it didn't seem like the spec mentioned what the `Promise` should be rejected with.

In the chrome examples it seems like they reject with a proper error:

```js
navigator.clipboard.writeText('Text to be copied')
  .then(() => {
    console.log('Text copied to clipboard');
  })
  .catch(err => {
    // This can happen if the user denies clipboard permissions:
    console.error('Could not copy text: ', err);
  });
```

ref: https://developers.google.com/web/updates/2018/03/clipboardapi#copy_writing_text_to_the_clipboard